### ISSUE TITLE
Use DANN with target features

### ISSUE THREAD

**Issue by rtaiello** [KevinMusgrave/pytorch-adapt#95]
Hi @KevinMusgrave,

How could I use DANN with target features, I tried to do that:
```
from pytorch_adapt.hooks import DANNHook, CLossHook, FeaturesAndLogitsHook
G.count, C.count, D.count = 0, 0, 0
f_hook =  FeaturesAndLogitsHook(domains = ["src", "target"])
c_hook = CLossHook(f_hook=f_hook)
hook = DANNHook(opts)
model_counts = validate_hook(hook, list(data.keys()))
outputs, losses = hook({**models, **data})
print_info(model_counts, outputs, losses, G, C, D)
```
But I'm still having:
```
Expected model counts = {'G': 2, 'D': 2, 'C': 1}
True model counts = G: 2, C: 1, D: 2
```
Thanks in advance!

**Comment by KevinMusgrave** [comment]
I pushed a change to the dev branch:
```
pip install pytorch-adapt==0.0.82.dev0
```

Now your code should work.

You can also pass in `domains` directly to `CLossHook`:
```python
c_hook = CLossHook(domains=["src", "target"])
```

The reason why your code broke is that `CLossHook` was hard-coded to expect "src" logits only. So when `FeaturesAndLogitsHook` returned two tensors for logits, it was "too many values to unpack".

Let me know if the latest version works and I'll push it to `0.0.82`.
